subsys/pm/power.c

```diff
@@ -122,29 +122,33 @@ static int pm_suspend_devices(void)
 
 	for (const struct device *dev = devs + devc - 1; dev >= devs; dev--) {
 		int ret;
+		struct pm_device *pm = dev->pm;
 
 		/*
-		 * ignore busy devices, wake up source and devices with
-		 * runtime PM enabled.
+		 * skip suspension if device:
+		 * - does not support PM
+		 * - is already suspended or turned off
+		 * - is busy
+		 * - is a wake up source
```

Comment:
"is an active(/enabled?) wake up source"

Reply:
adjusted comment